setup.cfg

```diff
@@ -1,5 +1,62 @@
 [metadata]
-description-file = README.md
+url = https://github.com/nipype/pydra
+author = nipype developers 
+author_email = neuroimaging@python.org
+maintainer = nipype developers
+maintainer_email = neuroimaging@python.org
+description = Pydra dataflow engine
+long_description = file:long_description.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+license = Apache License, 2.0
+provides =
+    pydra
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Operating System :: MacOS :: MacOS X
```

Comment:
We aren't testing on OSX. Probably should if we want to make this claim.

Reply:
we probably should add to travis. Of course, it is tested on my osx, but nothing more

Reply:
Okay. That feels like a lot to put in this already kind of big PR. I'll open an issue.

Reply:
we should i think also add a windows build via appveyor.

Reply:
Given how obnoxiously slow Appveyor is, I would propose giving Azure a shot.

Reply:
@effigies - the addition of testing on windows. @djarecka - we could ask hoda to look into this as part of getting her feet wet with all the tools.

Reply:
Created #82 and #83.

Reply:
you mean to install and test on her os, or adding windows to CI?

Reply:
both can be separately added based on the issues you created.